### PR TITLE
Add ap-southeast-2 to aws region specific endpoints

### PR DESCRIPTION
**Pull request by poornas**

### CODE REVIEW

File: minio/api.py [hunk 1]
```diff
@@ -219,9 +219,10 @@ def make_bucket(self, bucket_name, location='us-east-1'):
         Make a new bucket on the server.
 
         Optionally include Location.
-           ['us-east-1', 'us-west-1', 'us-west-2', 'eu-west-1', 'eu-central-1',
-            'ap-southeast-1', 'ap-southeast-2', 'ap-northeast-1', 'sa-east-1',
-            'cn-north-1']
+           ['us-east-1', 'us-east-2', 'us-west-1', 'us-west-2', 'eu-west-1',
+            'eu-west-2','ca-central-1', 'eu-central-1', 'sa-east-1',
```
Comment (harshavardhana) [inline minio/api.py]: A space missing `'eu-west-2',  'ca-central-1'`

File: minio/api.py [hunk 2]
```diff
@@ -219,9 +219,10 @@ def make_bucket(self, bucket_name, location='us-east-1'):
...
+            'cn-north-1','ap-southeast-1', 'ap-southeast-2',
```
Comment (harshavardhana) [inline minio/api.py]: Same here

File: minio/api.py [hunk 3]
```diff
@@ -219,9 +219,10 @@ def make_bucket(self, bucket_name, location='us-east-1'):
...
+            'ap-northeast-1','ap-northeast-2']
```
Comment (harshavardhana) [inline minio/api.py]: Same here

Comment (poornas) [reply minio/api.py]: done

**Pull request merged**